com.unity.shadergraph/Editor/Drawing/Views/PropertyNodeView.cs

```diff
@@ -42,6 +58,181 @@ public PropertyNodeView(PropertyNode node, EdgeConnectorListener edgeConnectorLi
 
         public VisualElement colorElement => null;
 
+        public string inspectorTitle
+        {
+            get
+            {
+                return $"{property?.displayName} (Node)";
```

Comment:
I'm not sure what  null property will do here - say "null"? Or empty string? Probably should handle it more explicitly, or just not handle it since a null property in a property node view shouldn't happen, right?

Reply:
Referencing this SO answer: https://stackoverflow.com/questions/35895250/how-are-null-values-in-c-sharp-string-interpolation-handled

If the property is null, then `null` is handed to the string interpolation, which replaces that with an empty string.

So it'll basically just be displayed as "  Node".

But a null property shouldn't be happening in a node view anyway so I think I can remove this check.